src/btree/bt_cursor.c

```diff
@@ -65,8 +72,6 @@ __btcur_bounds_search_near_reposition(WT_SESSION_IMPL *session, WT_CURSOR_BTREE
     key_out_of_bounds = upper = false;
 
     WT_ASSERT(session, WT_CURSOR_BOUNDS_SET(cursor));
-    if (CUR2BT(cbt)->type != BTREE_ROW)
```

Comment:
Given that we removed this check entirely does the code work for FLCS?

Reply:
No, I have added a check we can't set bounds on a btree if it is FLCS